lib/shared/bucketing-assembly-script/__tests__/types/configBody.test.ts

```diff
@@ -6,6 +6,8 @@ import {
     hasConfigDataForEtag,
 } from '../bucketingImportHelper'
 import cloneDeep from 'lodash/cloneDeep'
+import immutable from 'object-path-immutable'
```

Comment:
what does this library give us? is it just a quick way to generate unique + immutable configs depending on what's passed to the method?

Reply:
yeah its just a more ergonomic way to immutably modify deeply nested things in an object without having to do many levels of `...` spreading and stuff

Reply:
only using it for tests

Reply:
ah gotcha.. nice :) this is a much cleaner option!